### PR TITLE
Release Google.Cloud.BinaryAuthorization.V1Beta1 version 1.0.0-beta01

### PR DESCRIPTION
**Pull request by jskeet**
Changes in this release:

Intial beta release.

### CODE REVIEW

File: apis/apis.json [hunk 1]
```diff
@@ -299,6 +299,24 @@
         "Billing"
       ]
     },
+    {
+      "id": "Google.Cloud.BinaryAuthorization.V1Beta1",
+      "version": "1.0.0-beta01",
+      "type": "grpc",
+      "productName": "Binary Authorization API",
+      "productUrl": "https://cloud.google.com/binary-authorization/docs/reference/rpc",
+      "description": "The management interface for Binary Authorization, a system providing policy control for images deployed to Kubernetes Engine clusters.",
```
Comment (chrisdunelm) [inline apis/apis.json:308]: Often this is phrased "Recommended Google client library..." - should this also be phrased this way?

**Review by chrisdunelm — APPROVED** [review]
LGTM - one query

File: apis/apis.json (hunk 1 above)
Comment (jskeet) [reply apis/apis.json:308]: Yes, good call - will fix in the various commits.

Comment (jskeet) [reply apis/apis.json:308]: Fixed but then pushed to the wrong branch. So beta01 will have the suboptimal description - I'll create a new PR just to fix it for future releases.

**Pull request merged**